src/unwinder/i386/unwinder_i386.cpp

```diff
@@ -97,6 +97,10 @@ OOPStackUnwinderX86::VirtualUnwind(
 
     ContextRecord->ContextFlags |= CONTEXT_UNWOUND_TO_CALL;
 
+#define CALLER_SAVED_REGISTER(reg) if (rd.pCurrentContextPointers->reg) ContextRecord->reg = *rd.pCurrentContextPointers->reg;
```

Comment:
``UnwindEbpDoubleAlignFrame`` sets Eax, Ecx, Edx for debugging purpose:
```
3832 #ifdef _DEBUG
3833             /* The filter has to be called by the VM. So we dont need to
3834                update callee-saved registers.
3835              */
3836
3837             if (flags & UpdateAllRegs)
3838             {
3839                 static DWORD s_badData = 0xDEADBEEF;
3840
3841                 pContext->SetEaxLocation(&s_badData);
3842                 pContext->SetEcxLocation(&s_badData);
3843                 pContext->SetEdxLocation(&s_badData);
3844
3845                 pContext->SetEbxLocation(&s_badData);
3846                 pContext->SetEsiLocation(&s_badData);
3847                 pContext->SetEdiLocation(&s_badData);
3848             }
3849 #endif
```
I'm not sure which one is better: enclosing this region with _DEBUG, or simply removing. Please let me know your opinion.